bitnami/jupyterhub/templates/hub/service-account.yaml

```
@@ -0,0 +1,15 @@
+{{- if .Values.hub.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: hub
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  name: {{ template "jupyterhub.hubServiceAccountName" . }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+  namespace: {{ .Release.Namespace }}
```

Quote the namespace + add image pull secrets to SA:
```suggestion
  namespace: {{ .Release.Namespace | quote }}
{{- include "jupyterhub.imagePullSecrets" . }}
```

---

Done